components/concept/engine/src/main/java/mozilla/components/concept/engine/Settings.kt

```diff
@@ -174,6 +174,11 @@ abstract class Settings {
      * Setting to force the ability to scale the content
      */
     open var forceUserScalableContent: Boolean by UnsupportedSetting()
+
+    /**
+     * Setting to control clear color of GeckoView
```

Comment:
nit: On the concept level we should try to stay engine-agnostic :). Maybe just change the comment to "Engine View", or ideally describe what the clear color is?